src/ripple/rpc/handlers/AccountLines.cpp

```diff
@@ -97,13 +97,8 @@ Json::Value doAccountLines (RPC::Context& context)
     std::string strIdent (params[jss::account].asString ());
     AccountID accountID;
 
-    if (auto jv = RPC::accountFromString (accountID, strIdent))
-    {
-        for (auto it = jv.begin (); it != jv.end (); ++it)
-            result[it.memberName ()] = it.key ();
-
+    if ((result = RPC::accountFromString (accountID, strIdent)))
```

Comment:
Should we consider disabling the warning? Assignment in conditional when introducing a scoped variable seems to be getting popular.